From a496af6374060443468dabcbea10010487792915 Mon Sep 17 00:00:00 2001
From: kfx <kfx@rock-chips.com>
Date: Tue, 9 Jul 2013 10:55:43 +0800
Subject: [PATCH] Revert "sdmmc: module_init->fs_initcall, emmc:
 fs_inicall-->fs_initcall_sync"

This reverts commit aa4812bdf22211b0d9d2006cafad1ef40aa8aefe.
---
 drivers/mmc/host/rk29_sdmmc.c | 3 +--
 drivers/mmc/host/rkemmc.c     | 2 +-
 2 files changed, 2 insertions(+), 3 deletions(-)

diff --git a/drivers/mmc/host/rk29_sdmmc.c b/drivers/mmc/host/rk29_sdmmc.c
index 31408cf7d0f0..6f9280097345 100755
--- a/drivers/mmc/host/rk29_sdmmc.c
+++ b/drivers/mmc/host/rk29_sdmmc.c
@@ -4166,8 +4166,7 @@ static void __exit rk29_sdmmc_exit(void)
 	platform_driver_unregister(&rk29_sdmmc_driver);
 }
 
-//module_init(rk29_sdmmc_init);
-fs_initcall(rk29_sdmmc_init);
+module_init(rk29_sdmmc_init);
 module_exit(rk29_sdmmc_exit);
 
 MODULE_DESCRIPTION("Rk29 Multimedia Card Interface driver");
diff --git a/drivers/mmc/host/rkemmc.c b/drivers/mmc/host/rkemmc.c
index e567b56fbca3..6d195124c10f 100644
--- a/drivers/mmc/host/rkemmc.c
+++ b/drivers/mmc/host/rkemmc.c
@@ -1490,5 +1490,5 @@ static void __exit rk_mmc_exit(void)
 	platform_driver_unregister(&rk_mmc_driver);
 }
 
-fs_initcall_sync(rk_mmc_init);
+fs_initcall(rk_mmc_init);
 module_exit(rk_mmc_exit);
-- 
2.35.3

